content/en/docs/install/rancher-desktop.md

```diff
@@ -20,36 +20,29 @@ essentials to work with containers and Kubernetes on your desktop.
 
   1. **Download Rancher Desktop**:
       - Navigate to the [Rancher Desktop releases
-        page](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.14.0).
-      - Select the appropriate installer for your operating system for version 1.14.0.
+        page](https://github.com/rancher-sandbox/rancher-desktop/releases).
```

Comment:
Is it worth putting a note somewhere on the page along the lines of "we tested this with Rancher Desktop 1.2.3 but it should work with more recent versions" and maybe inviting readers to raise an issue if it doesn't?